tests/test.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: Test sansible.vernemq Role
+  hosts: all
+
+  vars:
+    vernemq:
```

Comment:
Should be sansible_vernemq.

Reply:
Moved the variables to the role call directly.